pom.xml

```diff
@@ -235,6 +235,11 @@
         <artifactId>protobuf-java</artifactId>
         <version>3.7.1</version>
       </dependency>
+      <dependency>
+        <groupId>com.lmax</groupId>
+        <artifactId>disruptor</artifactId>
+        <version>3.4.2</version>
+      </dependency>
```

Comment:
This looks like a new dependency.  Is this needed by log4j2?  What uses this?

Reply:
So, this is used as an optional dependency of log4j2, that enables the high-performance asynchronous loggers. It is required on the class path if `-Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector` system property is set. I set it by default in our example `accumulo-env.sh`, because there's no good reason to not have it, since it's ALv2 licensed has no deps.

Reply:
The LMAX disrupter is basically a ring buffer that is designed to minimize locking and from most reports significantly improves performance. We should use it if at all possible.  Enabling it and keeping the dependency seems to make it easier on users.

Reply:
The LMAX disrupter is basically a ring buffer that is designed to minimize locking and from most reports significantly improves performance. We should use it if at all possible.  Enabling it and keeping the dependency seems to make it easier on users and should help to encourage its usage.